recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "jellyfish" %}
-{% set version = "0.7.2" %}
+{% set version = "0.8.2" %}
 {% set bundle = "tar.gz" %}
-{% set hash = "cb09c50d7e2bb7b926fc7654762bc81f9c629e0c92ae7137bf22b34f39515286" %}
+{% set hash = "a499741401512d05bbd3556e448e960bc908eba3879fb73d450e8e91566a030b" %}
```

Comment:
In the future I recommend using grayskull to regenerate the whole recipe and "modernize" all these jinja bits that could be troublesome.

Reply:
yes, I've started doing that routinely now. This PR is from before that time 🙂